build.gradle

```diff
@@ -19,6 +19,14 @@ allprojects {
     }
 }
 
+ext {
+    compileSdkVersion = 25
+    buildToolsVersion = '25.0.2'
+    minSdkVersion = 16
+    targetSdkVersion = compileSdkVersion
+
+    supportLibVersion = '25.3.0'
```

Comment:
Similar to the above comment, you are updating the support lib version here, which should be done in separate commit. This commit should only be a refactor diff, putting out some of the variables in top level gradle file